enclave-tls/samples/enclave-tls-client/client.c

```diff
@@ -234,41 +288,16 @@ int main(int argc, char **argv)
 			     "        --log-level/-l        set the log level\n"
 			     "        --ip/-i               set the listening ip address\n"
 			     "        --port/-p             set the listening tcp port\n"
-			     "        --product-enclave/-P  set to enable product enclave\n"
+			     "        --debug-enclave/-D    set to enable enclave debugging\n"
 			     "        --help/-h             show the usage\n");
+			exit(1);
+			/* Avoid compiling warning */
+			break;
```

Comment:
Removing above 2 lines still causes build warning?

Reply:
yes,   in switch context, it must have "break". Otherwise it will have build warning